packages/hash/frontend/src/pages/entity-editor.page.tsx

```diff
@@ -157,11 +159,15 @@ const ExampleEntityEditorPage: NextPageWithLayout = () => {
     }
   }, [loadingUser, router, kratosSession]);
 
-  return loadingUser || !authenticatedUser || loadingTypeSystem ? (
```

Comment:
Loading user seemed to always be true, I'm not sure why.. we probably want to look into this